projects/fuzztest-example/setup_configs.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -eu
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+echo "
```

Comment:
We will eventually move these to https://github.com/google/fuzztest/blob/main/bazel/setup_configs.sh, right?

Reply:
Yep!

Reply:
Put up a PR: https://github.com/google/fuzztest/pull/69